components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -100,6 +106,22 @@ function GetBackgroundImageSrc (props: Props) {
   return undefined
 }
 
+function GetSponsoredRichMediaBackground(props: Props): SponsoredRichMediaBackgroundInfo | undefined {
+  const wallpaperData = props.newTabData.brandedWallpaper
+  if (!props.newTabData.showBackgroundImage
```

Comment:
[nit] It's a bit challenging to read intent from this long conditional expression. One option to improve readability might be to store the value in a variable name that captures intent:

```ts
const showRichMediaBackground =
  props.newTabData.showBackgroundImage &&
  wallpaperData &&
  wallpaperData.isSponsored &&
  wallpaperData.type === 'richMedia' &&
  wallpaperData.wallpaperImageUrl

if (!showRichMediaBackground) {
  return
}

```

Reply:
Fixed, thanks